### PR TITLE
Entire Site Overhaul

### PR DESCRIPTION
**Pull request by foopod**
Reasons for overhaul

+ Move from Single Page Site to Multiple Pages with the bulk of content in markdown to make contributions easier.
+ Updated information about space and tools, new photos
+ Simple, more welcoming design
+ Blogging functionality if we want to use it

### CODE REVIEW

**Comment by foopod** [comment]
This is an entire rewrite. So rather than looking at the diffs, it might be easier to checkout the two versions to compare. There is also a live preview of the new site at https://hackland.netlify.app/.

File: tools.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        <p>
+            Fully dust extracted. Induction required to use!
+        </p>
+        <p>
+            Courses running for use (open to public) - register: <a href="https://www.meetup.com/HackLand/">https://www.meetup.com/HackLand/</a>
+        </p>
+    </div>
+</div>
+
+<div class="tool">
```
Comment (acreegan) [inline tools.md:33]: I would suggest that all photos in this section be rotated so that their orientation is correct

File: join.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+description: 
+---
+
+Members can use the space whenever they want, 24 hours every day of the week.
+
+The hackspace is regularly open to the public whenever a member is in to open up, and weekly on Thursday from 6pm - this is the busiest time and the best for new people visiting.
+
+Membership costs $20 a week but please pay as much as you feel is fair so the space can survive.
+
+Membership includes full access to everything 24/7* including:
```
Comment (acreegan) [inline join.md]: Suggest putting a * where the footnote/conditions to the 24/7 are written. Is it the Other things to note section?

File: join.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
++ UNLIMITED Fibre Internet
++ Access to all of the tools
++ A Storage Box for projects
++ A quiet place to work (sometimes quiet)
++ All of the tea / coffee
+
+Other Things to note:
+
++ No powertools/noisey things after 10pm
```
Comment (acreegan) [inline join.md]: noisey -> noisy

File: tools.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    </div>
+</div>
+
+
+<div class="tool">
+    <img class="tool-pic" src="/public/images/tools/shadow-board.jpg"/>
+    <div class="tool-description">
+        <h3 class="tool-title">Hand Tools</h3>
+        <p>
+            For woodword, metalwork, randomwork and much more.
```
Comment (acreegan) [inline tools.md]: woodward -> woodwork

**Review by acreegan — APPROVED** [review]
New site working fine for me. Tested on Chrome and Chrome for Android

File: tools.md (hunk 1 above)
Comment (foopod) [reply tools.md:33]: Sigh, just checked chrome. This isn't a problem in Firefox. There must be meta data in the images that only certain browsers use. Thanks!

File: join.md (hunk 2 above)
Comment (foopod) [reply join.md]: Fixed

File: join.md (hunk 3 above)
Comment (foopod) [reply join.md]: Fixed

File: tools.md (hunk 4 above)
Comment (foopod) [reply tools.md]: fixed

**Pull request merged**